examples/guides/se-restful-webservice/README.adoc

```diff
@@ -70,135 +73,78 @@ See the <<downloading,downloading>> section for instructions.
-        <copied.libs.dir>${project.build.directory}/${libs.classpath.prefix}</copied.libs.dir>
-        ...
-    </properties>
-
-    <build>
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-jar-plugin</artifactId>
-                    <version>2.5</version>
-                    <configuration>  <!--2-->
-                        <archive>
-                            <manifest>
-                                <addClasspath>true</addClasspath>
-                                <classpathPrefix>${libs.classpath.prefix}</classpathPrefix>
-                                <mainClass>${mainClass}</mainClass>
-                            </manifest>
-                        </archive>
-                    </configuration>
-                </plugin>
-            </plugins>
-        </pluginManagement>
-        <plugins>
-            <plugin> <!--3-->
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>copy-dependencies</id>
-                        <phase>prepare-package</phase>
-                        <goals>
-                            <goal>copy-dependencies</goal>
-                        </goals>
-                        <configuration>
-                            <outputDirectory>${copied.libs.dir}</outputDirectory>
-                            <overWriteReleases>false</overWriteReleases>
-                            <overWriteSnapshots>false</overWriteSnapshots>
-                            <overWriteIfNewer>true</overWriteIfNewer>
-                            <overWriteIfNewer>true</overWriteIfNewer>
-                            <includeScope>runtime</includeScope>
-                            <excludeScope>test</excludeScope>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
-----
-<1> Make sure to specify your own main class path.
-<2> Instructs Maven what main class to set in the JAR's manifest and what prefix to use for
-copied dependency JARs.
-<3> Tells Maven to package the dependency JARs in the specified directory relative to the project's JAR.
-// end::runMavenOutsideIDE[]
-
-=== Create a config resource file
-Your app will use the Helidon config support to initialize the greeting and set up HTTP
-listening.
+// _include::0-3:{pom}[tag=configYamlDependency,indent=0]
+----
+<dependency>
+    <groupId>io.helidon.config</groupId>
+    <artifactId>helidon-config-yaml</artifactId>
+</dependency>
+----
 
-Create this config file:
+==== `src/main/resources/application.yaml`: a config resource file for the application
+Your app uses Helidon config to initialize the greeting and set up HTTP
+listening.
```

Comment:
Could we change the last phrase to: "...and set up the HTTP listener." ?

"HTTP listening" is an unusual term, for me.

Reply:
Done